testdata/mongodb_exporter.testFlagHelp.golden

```diff
@@ -1,53 +1,57 @@
+      --web.metrics-path="/metrics"  
+                                 Path under which to expose metrics.
+      --collect.database         Enable collection of Database metrics
+      --collect.collection       Enable collection of Collection metrics
+      --collect.topmetrics       Enable collection of table top metrics
+      --collect.indexusage       Enable collection of per index usage stats
+      --mongodb.uri=[mongodb://][user:pass@]host1[:port1][,host2[:port2],...][/database][?options]  
+                                 MongoDB URI, format
+      --mongodb.tls              Enable tls connection with mongo server
+      --mongodb.tls-cert=""      Path to PEM file that contains the certificate
+                                 (and optionally also the decrypted private key
+                                 in PEM format).
+                                 
+                                   This should include the whole certificate chain.
+                                   If provided: The connection will be opened via TLS to the MongoDB server.
+      --mongodb.tls-private-key=""  
+                                 Path to PEM file that contains the decrypted
+                                 private key (if not contained in
+                                 mongodb.tls-cert file).
+      --mongodb.tls-ca=""        Path to PEM file that contains the CAs that are
+                                 trusted for server connections.
+                                 
+                                   If provided: MongoDB servers connecting to should present a certificate signed by one of this CAs.
+                                   If not provided: System default CAs are used.
+      --mongodb.tls-disable-hostname-validation  
+                                 Disable hostname validation for server
+                                 connection.
+      --mongodb.max-connections=1  
+                                 Max number of pooled connections to the
+                                 database.
+      --test                     Check MongoDB connection, print buildInfo()
+                                 information and exit.
+      --mongodb.socket-timeout=3s  
+                                 Amount of time to wait for a non-responding
+                                 socket to the database before it is forcefully
+                                 closed.
+                                 
+                                   Valid time units are 'ns', 'us' (or 'µs'), 'ms', 's', 'm', 'h'.
+      --mongodb.sync-timeout=1m  Amount of time an operation with this session
+                                 will wait before returning an error in case
+                                 
+                                   a connection to a usable server can't be established.
+                                   Valid time units are 'ns', 'us' (or 'µs'), 'ms', 's', 'm', 'h'.
+      --groups.enabled=""        Currently ignored
+
```

Comment:
I honestly do not know if we need that test at all…

Reply:
Why not? It works fine and it helps to prevent accidental flags change.

Reply:
Because you don't actually check that – you just copy the actual output to the golden file. But that's up to you.

Reply:
I'll make a note and try to figure out how to improve the test in future, but I'm going to keep it for now.
Btw it's hard to change the `.golden` file by hand as output length of the string may differ between your terminal and test output. So you should run `go test ./. -update` manually to change the file. It you forget to do this, tests will fail.